### PR TITLE
updating benchmarks for js framework

### PR DESCRIPTION
**Pull request by farooqkz**

### CODE REVIEW

**Comment by farooqkz** [comment]
ping @Havunen

File: src/components/Benchmarks.js [hunk 1]
```diff
@@ -79,38 +79,37 @@ export default class Benchmarks extends Component {
           data: [
             {
               label: 'Vanilla JS',
-              score: 1.0
+              score: 1.02
             },
             {
-              label: 'Inferno 5.3.0',
```
Comment (Havunen) [inline src/components/Benchmarks.js:85]: From where are these numbers coming? And what do they mean?

Comment (farooqkz) [reply src/components/Benchmarks.js:85]: From here:
https://github.com/krausest/js-framework-benchmark/

Comment (Havunen) [reply src/components/Benchmarks.js:85]: Ok, fine :)

**Comment by Havunen** [comment]
Thanks for the PR!

**Pull request merged**